### PR TITLE
Update rules

### PR DESCRIPTION
**Pull request by Scrumplex**

### CODE REVIEW

File: src/welcome-channel.yaml [hunk 1]
```diff
@@ -15,17 +15,11 @@
...
-    Keep all activity legal, legitimate, and appropriate: We do not tolerate users breaking any laws, breaching a Terms of Service or an EULA, sexual content, or anything the moderators determine to be malicious or inappropriate.
-    Don't spam and don't scam: No advertisement outside of the showcase channels. Also, no NFTs.
-    Follow the given channel topic: Don't derail a discussion, and don't start conversations outside of the channel in which they're most appropriate. For example, if you start to get off-topic in an on-topic channel, ask other participants to move to an off-topic channel.
-    When requesting help, specify all relevant information and provide error logs: Help us to help you! Without this information, we won't be able to help with your issue as easily or as quickly.
+    Do not harass anyone: We want to make participation in our community a harassment-free experience for everyone. If you think that someone is breaking the rules, contact the moderators, instead of harassing them.
```
Comment (flowln) [inline src/welcome-channel.yaml:18]: Maybe you could mention ModMail here? :thinking:

**Comment by NotJansel** [comment]
Can we please also add a Rule to not ping moderators when requesting support? it has happened multiple times already that every moderator got pinged in a support thread which is not related to the community itself

File: src/welcome-channel.yaml [hunk 2]
```diff
@@ -3,29 +3,24 @@
...
+    Keep all activity legal, legitimate, and appropriate: We do not tolerate users breaking any laws, breaching a ToS or an EULA, posting sexual content, or anything the moderators determine to be malicious or inappropriate. Shilling crypto currency related products and gambling websites is strictly forbidden.
+    Follow the given channel topic: Don't derail a discussion, and don't start conversations outside of the channel in which they're most appropriate. For example, if you start to get off-topic in an on-topic channel, ask other participants to move to an off-topic channels. Advertisements are not allowed outside of showcase channels.
     Don't import drama or controversy from other communities: In these topics, emotions run high and opinions run even higher. To avoid arguments and other similar problems, simply avoid discussion of controversial topics from other communities.
-    Leave other launcher developers alone: We will not tolerate bashing on the developers of other launchers, including those which we have forked from.
```
Comment (Akarys42) [inline src/welcome-channel.yaml:23]: That one should be kept

**Review by Akarys42 — CHANGES_REQUESTED** [review]

**Comment by IThundxr** [comment]
Looks good to me :)

File: src/welcome-channel.yaml (hunk 2 above)
Comment (Scrumplex) [reply src/welcome-channel.yaml:23]: I thought this is already implied with the "Do not harass anyone" rule. Maybe I can add another sentence about this there.

**Review by Akarys42 — APPROVED** [review]

**Review by Regular-Baf — APPROVED** [review]

**Review by NotJansel — APPROVED** [review]

**Review by ryanccn — APPROVED** [review]

**Pull request merged**